### PR TITLE
chore: tests for adding map layers

### PR DESCRIPTION
**Pull request by jenniferarnesen**
Test adding map layers. It does not yet test the actual rendered map.

The files in the /elements folder are a first attempt at page objects. They are classes that hide complexity of interacting with the page so that the actual tests in the /integration folder read nicely. Most methods in the page objects return `this` so that the tests can chain the methods.

I'm not generally a big fan of complex class inheritance, but in the case of the page object layers, it made sense to me. Open to other ideas though.

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (martinkrulltott) [inline cypress/fixtures/example.json:1]: remove this file?

Comment (jenniferarnesen) [reply cypress/fixtures/example.json]: Removed

File: cypress/integration/layers/boundarylayer.cy.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    it('shows error if no orgunit selected', () => {
+        Layer.openDialog('Boundaries').addToMap();
+
+        Layer.validateDialogClosed(false);
+
+        cy.contains('No organisation units are selected').should('be.visible');
+    });
+
+    it('adds a boundary layer', () => {
+        Layer.openDialog('Boundaries')
```
[the remark this answers is not in the document]
Comment (jenniferarnesen) [reply cypress/integration/layers/boundarylayer.cy.js:19]: As agreed on slack, let's give page objects a try so we can compare to Cucumber and the DV "api" approach.

File: cypress/elements/layer.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    validateDialogClosed(closed) {
+        if (closed) {
+            cy.get('[data-test="dhis2-uicore-card"]').should('not.exist');
+        } else {
+            cy.get('[data-test="dhis2-uicore-card"]').should('be.visible');
+        }
+    }
+
+    validateCard(title, items) {
```
[the remark this answers is not in the document]
Comment (jenniferarnesen) [reply cypress/elements/layer.js]: Changed to `validateCardTitle` and `validateCardItems`

**Review by turban — APPROVED** [review]

**Pull request merged**